contracts/SBCDepositContract.sol

```diff
@@ -264,12 +265,11 @@ contract SBCDepositContract is
      * NOTE: This function signature is hardcoded in the Gnosis execution layer clients. Changing this signature without updating the
      * clients will cause block verification of any post-shangai block to fail. The function signature cannonical spec is here
      * https://github.com/gnosischain/specs/blob/master/execution/withdrawals.md
-     * @param _deprecatedUnused Previously `maxFailedWithdrawalsToProcess` currently deprecated and ignored
      * @param _amounts Array of amounts to be withdrawn.
      * @param _addresses Array of addresses that should receive the corresponding amount of tokens.
      */
     function executeSystemWithdrawals(
-        uint256 _deprecatedUnused,
+        uint256 /* _deprecatedUnused */,
```

Comment:
Unrelated diff, please revert lines 267 and 272

Reply:
Well, maybe I should rename the PR to "fix all solhint issues", which this does. Or do you still prefer reverting that?

Reply:
Oh if solhint requires that change then all good :+1:

Reply:
It does:
```
,Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
   --> project:/contracts/SBCDepositContract.sol:273:9:
    |
273 |         uint256 _deprecatedUnused,
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^
```